tests/Doctrine/Tests/ORM/Query/QueryTest.php

```diff
@@ -175,6 +176,39 @@ public function testCollectionParameters()
         $this->assertEquals($cities, $parameter->getValue());
     }
 
+    public function provideProcessParameterValueIterable() : Generator
+    {
+        $baseArray = [
+            0 => 'Paris',
+            3 => 'Canne',
```

Comment:
```suggestion
            3 => 'Cannes',
```

Reply:
😁 It was code that was already in the repository but I confess I didn't notice the type. Fixed :)